Tools/README

```diff
@@ -1,7 +1,7 @@
 This directory contains a number of Python programs that are useful
 while building or extending Python.
 
-buildbot        Batchfiles for running on Windows buildslaves.
```

Comment:
Interesting, "buildslave" was a term used in the Buildbot documentation: http://docs.buildbot.net/0.8.5/manual/cfg-buildslaves.html. In recent versions it was replaced with "worker": http://docs.buildbot.net/1.4.0/manual/cfg-workers.html.